android/assets/jsons/translations/Portuguese.properties

```diff
@@ -99,12 +97,9 @@ We will remember this. = Nós nos lembraremos disto.
 [civName] and [targetCivName] have signed a Peace Treaty! = [civName] e [targetCivName] assinaram um Tratado de Paz!
 [civName] and [targetCivName] have signed the Declaration of Friendship! = [civName] e [targetCivName] assinaram a Declaração de Amizade!
 [civName] has denounced [targetCivName]! = [civName] condenou [targetCivName]!
- # Requires translation!
-Do you want to break your promise to [leaderName]? = 
- # Requires translation!
-We promised not to settle near them ([count] turns remaining) = 
- # Requires translation!
-They promised not to settle near us ([count] turns remaining) = 
+Do you want to break your promise to [leaderName]? = Queres quebrar a tua promessa para com [leaderName]?
+We promised not to settle near them ([count] turns remaining) = Prometemos não nos acomodarmos perto deles ([count] turnos restantes)
+They promised not to settle near us ([count] turns remaining) = Eles prometeram não se estabelecerem perto de nós ([contagem] turnos restantes)
```

Comment:
For example here you translated "[count]" to "[contagem]" while you shouldn't translate it. Other errors seems to be of the same kind.

Reply:
Replace those translated labels with their original english names

Reply:
ok. I will do this